lib/PICA/Path.pm

```diff
@@ -261,6 +447,270 @@ without inspection field and occurrence values.
 Stringifies the PICA path to normalized form. Subfields are separated with
 C<$>, unless called as C<stringify(1)> or the first subfield is C<$>.
 
+=head1 Matching rules
+
+    ...
```

Comment:
~~~
=head1 MATCHING RULES

Matching rules are illustrated with the following sample record:
~~~

And link to this section from INTRODUCTION above or move the whole section into INTRODUCTION